src/test/java/com/exasol/adapter/dynamodb/mapping/AbstractValueMapperTest.java

```diff
@@ -0,0 +1,90 @@
+import com.exasol.dynamodb.resultwalker.IdentityDynamodbResultWalker;
+import com.exasol.dynamodb.resultwalker.ObjectDynamodbResultWalker;
+import com.exasol.sql.expression.StringLiteral;
+import com.exasol.sql.expression.ValueExpression;
+
+/**
+ * Tests for {@link AbstractValueMapper}
+ */
+public class AbstractValueMapperTest {
+    @Test
+    void testLookup() throws DynamodbResultWalkerException, ValueMapperException {
+        final ObjectDynamodbResultWalker resultWalker = new ObjectDynamodbResultWalker("isbn", null);
+        final MocColumnMappingDefinition columnMappingDefinition = new MocColumnMappingDefinition("d", resultWalker,
+                AbstractColumnMappingDefinition.LookupFailBehaviour.EXCEPTION);
+        final String isbn = "123456789";
+        final AttributeValue isbnValue = AttributeValueTestUtils.forString(isbn);
+        final ValueExpression valueExpression = new MocValueMapper(columnMappingDefinition)
+                .mapRow(Map.of("isbn", isbnValue));
+        assertThat(valueExpression.toString(), equalTo(isbn));
+    }
+
+    @Test
+    void testNullLookupFailBehaviour() throws ValueMapperException, DynamodbResultWalkerException {
+        final ObjectDynamodbResultWalker resultWalker = new ObjectDynamodbResultWalker("nonExistingColumn", null);
+        final MocColumnMappingDefinition columnMappingDefinition = new MocColumnMappingDefinition("d", resultWalker,
+                AbstractColumnMappingDefinition.LookupFailBehaviour.DEFAULT_VALUE);
+        final ValueExpression valueExpression = new MocValueMapper(columnMappingDefinition)
+                .mapRow(Collections.emptyMap());
+        assertThat(valueExpression.toString(), equalTo("default"));
+    }
+
+    @Test
+    void testExceptionLookupFailBehaviour() {
+        final ObjectDynamodbResultWalker resultWalker = new ObjectDynamodbResultWalker("nonExistingColumn", null);
+        final MocColumnMappingDefinition columnMappingDefinition = new MocColumnMappingDefinition("d", resultWalker,
+                AbstractColumnMappingDefinition.LookupFailBehaviour.EXCEPTION);
+        assertThrows(DynamodbResultWalkerException.class,
+                () -> new MocValueMapper(columnMappingDefinition).mapRow(Collections.emptyMap()));
+    }
+
+    @Test
+    public void testColumnMappingException() {
+        final String columnName = "name";
+        final MocColumnMappingDefinition mappingDefinition = new MocColumnMappingDefinition(columnName,
+                new IdentityDynamodbResultWalker(), AbstractColumnMappingDefinition.LookupFailBehaviour.EXCEPTION);
+        final ValueMapperException exception = assertThrows(ValueMapperException.class,
+                () -> new ExceptionMocValueMapper(mappingDefinition).mapRow(Map.of()));
+        assertThat(exception.getCausingColumn().getExasolName(), equalTo(columnName));
+    }
+
+    private static class MocValueMapper extends AbstractValueMapper {
```

Comment:
If you meant `Mock` here, please rename all to `MockValueMapper`. Same for all other classes starting with `Moc`.